README.md

```diff
@@ -1,16 +1,27 @@
-[![Build Status](https://api.travis-ci.org/47deg/scala-exercises.svg?branch=version-2)](https://travis-ci.org/47deg/scala-exercises?branch=version-2) [![Stories in Ready](https://badge.waffle.io/47deg/scala-exercises.png?label=ready&title=Ready)](https://waffle.io/47deg/scala-exercises)
+[![Stories in Ready](https://badge.waffle.io/47deg/scala-exercises.png?label=ready&title=Ready)](https://waffle.io/47deg/scala-exercises)
+[![Build status](https://badges.gitter.im/Join%20Chat.svg)](https://travis-ci.org/47deg/scala-exercises.svg?branch=master)
 
```

Comment:
It'd be nice to add a badge with a link to the Gitter channel. See the example in http://shields.io/